website/docs/docs/quickstarts/dbt-cloud/bigquery-qs.md

```diff
@@ -0,0 +1,186 @@
+---
+title: "Quickstart for dbt Cloud and BigQuery"
+description: "Quickstart for dbt Cloud and BigQuery."
+id: "bigquery"
+sidebar_label: "BigQuery quickstart"
+---
+For the BigQuery project in the quickstart guide, you'll learn how to set up BigQuery and connect it to dbt Cloud.
+This guide will walk you through:
+
+- Setting up a new BigQuery instance &mdash; Log in and create a new Google Cloud project​.
+- Accessing sample data in a public dataset &mdash; BigQuery supports public data sets that can be directly queried, so we will show you how to access those datasets via select statements. Additionally, we will show you how to populate your own database objects with that data.
+- Connecting dbt Cloud to BigQuery &mdash; 
+
+:::tip Videos for you
+You can check out [dbt Fundamentals](https://courses.getdbt.com/courses/fundamentals) for free if you're interested in course learning with videos.
+
+:::
+
+## Prerequisites​
+
+- You have a  [dbt Cloud account](https://cloud.getdbt.com/). 
+- You have a [Google account](https://support.google.com/accounts/answer/27441?hl=en).
+- You can use a personal or work account to set up BigQuery through [Google Cloud Platform (GCP)](https://cloud.google.com/free).
+
+## Create a new GCP project​
+
+1. Go to the [BigQuery Console](https://console.cloud.google.com/bigquery) after you log in to your Google account. If you have multiple Google accounts, make sure you’re using the correct one. 
+2. Create a [new project from the Console](https://cloud.google.com/resource-manager/docs/creating-managing-projects#creating_a_project). GCP automatically populates the Project name field for you. You can change it to be more descriptive for your use. For example, `dbt Learn - BigQuery Setup`.
+
+## Create BigQuery datasets
+
+1. Verify that you can run SQL queries. Copy and paste these queries into the Query Editor: 
```

Comment:
I'm still at the managed resources page. Maybe we should orient the user by adding a step:
From the [BigQuery console](https://console.cloud.google.com/bigquery), click **Editor**.

Reply:
We might also tell then to make sure they selected their newly created project, which is shown at the top.